lib/cambiatus_web/email.ex

```diff
@@ -21,55 +25,93 @@ defmodule CambiatusWeb.Email do
-      html_body: CambiatusWeb.EmailView.render("transfer.html", %{transfer: transfer})
-    )
+    new()
+    |> from({"#{community.name} - Cambiatus", Mailer.sender()})
+    |> to(recipient.email)
     |> set_language(transfer)
+    |> subject(gettext("You received a new transfer on") <> " #{community.name}")
+    |> render_body("transfer.html", render_params(transfer))
+    |> header("List-Unsubscribe", one_click_unsub(recipient, community, "transfer_notification"))
+    |> header("List-Unsubscribe-Post", "List-Unsubscribe=One-Click")
     |> Mailer.deliver()
   end
 
   def claim(claim) do
+    community = claim.action.objective.community
+    claimer = claim.claimer
+
     new()
-    |> from({"#{claim.action.objective.community.name} - Cambiatus", "no-reply@cambiatus.com"})
-    |> to(claim.claimer.email)
-    |> subject(gettext("Your claim was approved!"))
-    |> render_body("claim.html", %{claim: claim})
+    |> from({"#{community.name} - Cambiatus", Mailer.sender()})
+    |> to(claimer.email)
     |> set_language(claim)
+    |> subject(gettext("Your claim was approved!"))
+    |> header("List-Unsubscribe", one_click_unsub(claimer, community, "claim_notification"))
+    |> header("List-Unsubscribe-Post", "List-Unsubscribe=One-Click")
+    |> render_body("claim.html", render_params(claim))
     |> Mailer.deliver()
   end
 
   # input is a community with preloaded news with less than 30 days and members with active digest
   def monthly_digest(community) do
     Enum.each(community.members, fn member ->
       new()
-      |> from({"#{community.name} - Cambiatus", "no-reply@cambiatus.com"})
+      |> from({"#{community.name} - Cambiatus", Mailer.sender()})
       |> to(member.email)
       |> set_language(member.language)
       |> subject(gettext("Community News"))
-      |> render_body("monthly_digest.html", %{community: community, user: member})
+      |> header("List-Unsubscribe", one_click_unsub(member, community, "digest"))
+      |> header("List-Unsubscribe-Post", "List-Unsubscribe=One-Click")
+      |> render_body("monthly_digest.html", render_params(member, community))
       |> Mailer.deliver()
     end)
   end
 
+  defp render_params(%Transfer{} = transfer) do
+    unsub_link = unsub_link(transfer.to, transfer.community, transfer.to.language)
+    %{transfer: transfer, unsub_link: unsub_link}
+  end
+
+  defp render_params(%Claim{} = claim) do
+    unsub_link =
+      unsub_link(claim.claimer, claim.action.objective.community, claim.claimer.language)
+
+    %{claim: claim, unsub_link: unsub_link}
+  end
+
+  defp render_params(%User{} = user, %Community{} = community) do
+    unsub_link = unsub_link(user, community, user.language)
+    %{community: community, user: user, unsub_link: unsub_link}
+  end
+
+  def one_click_unsub(member, community, subject) do
+    token = AuthToken.sign(member, "email")
+
+    "<https://#{community.subdomain.name}/api/unsubscribe/#{subject}/#{token}>"
+  end
```

Comment:
Maybe it would make more sense to just return the url (without the brackets), and include the brackets in the calling code, to be a bit more reusable

```suggestion
  def one_click_unsub(member, community, subject) do
    token = AuthToken.sign(member, "email")

    "https://#{community.subdomain.name}/api/unsubscribe/#{subject}/#{token}"
  end
```

Reply:
+1